packages/react-hooks/src/hooks/debounced.ts

```diff
@@ -13,12 +13,12 @@ export function useDebouncedValue<T>(
   const [state, setState] = useState<T>(value);
 
   useEffect(() => {
-    const timeout = window.setTimeout(() => {
+    const timeout = setTimeout(() => {
```

Comment:
I also think we should only call this if `state` is different then `value. Otherwise, the will run even on the first render.